README.md

```diff
@@ -149,6 +149,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Issues**: [jabref](https://github.com/search?&q=user%3Ajabref+type:issue&state=open&type=Issues)
 - **Notes**: More details in [the blog post](https://www.jabref.org/hacktoberfest/2019.html).
 
+#### **Jenkins**
+
+- **Swag**: limited edition swag
+- **Requirements**: PR to any repositories under [any repo of Jenkins](https://issues.jenkins-ci.org/browse/JENKINS-59817?filter=-4). The PR must be at least raised in October 2019 and you should not be inactive for seven days or more during code review.
```

Comment:
Filter is incorrect, and  we do not guarantee swag for all contributors

Reply:
So far I can guarantee swag for Top 20 contributors, working on the budget things to get more